source/models/bounties/Bounty.js

```diff
@@ -0,0 +1,124 @@
+﻿const { EmbedBuilder, Guild } = require('discord.js');
+const { DataTypes: { BIGINT, STRING, INTEGER, BOOLEAN }, Model } = require('sequelize');
+const { ihpAuthorPayload } = require('../../embedHelpers');
+const { Hunter } = require('../users/Hunter');
+const { Guild: HunterGuild } = require('../guilds/Guild');
+const { database } = require('../../../database');
+
+/** Bounties are user created objectives for other server members to complete */
+const bountyModel = {
+	id: {
+		primaryKey: true,
+		type: BIGINT,
+		autoIncrement: true
+	},
```

Comment:
I know that we've talked about this at some point, but is there a reasonable future where the primary key is the combination of the user id, guild id, and slot number? that may be a bit much to have a 3-part primary key, especially if we persist this information in the database after completion, though.

Reply:
If we went with a composite key for Bounties, that means only current bounties would be allowed in the Bounties table (which would also mean we'd want another table for `completed|deleted` bounties). I think it's be more likely we'll want to have both current and historic bounties in the same table.